src/block/block_compact.c

```diff
@@ -67,7 +67,7 @@ __wt_block_compact_skip(WT_SESSION_IMPL *session, WT_BLOCK *block, int *skipp)
 	 * worth doing.  Ignore small files, and files where we are unlikely
 	 * to recover 10% of the file.
 	 */
-	if (fh->size <= 10 * 1024)
+	if (fh->size <= 10 * WT_MEGABYTE)
```

Comment:
Ten megabytes is as arbitrary as 10 K. We could move this test up into the btree layer and make it a page count, or we could make it configurable through the API.